app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_retired" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Retired Scenario</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario has been retired from organized play. It normally is not available to be played for credit,
+              but will be for this event, unless marked as for "No Society Credit".</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.core? %>
-    <span class="badge badge-core">Core</span>
+    <button type="button" class="btn btn-round-xs btn-core btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_core">Core</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_core" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Core Campaign</h4>
+          </div>
+          <div class="modal-body">
+            <p>This table is being offered as a Core Campaign game. You must have a Core Campaign character in order to
+              play, unless you would like play a Core pregen.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.non_pfs? %>
-    <span class="badge badge-non-pfs">No Society Credit</span>
+    <button type="button" class="btn btn-round-xs btn-non-pfs btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_nonpfs">No
+      Society Credit
+    </button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_nonpfs" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">No Society Credit</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario is not being run for organized play credit. You will not be receiving a chronicle for
+              playing. It may be pregen only, with Society characters, or otherwise. See "More Info!" for more
+              details.</p>
+          </div>
```

Comment:
Reword: This scenario is not being run for organized play credit - you will not receive a chronicle for playing. See "More Info!" for details on if characters or provided, or if you should bring your own.